config/deploy.rb

```diff
@@ -0,0 +1,47 @@
+# config valid only for current version of Capistrano
+lock '3.3.5'
+
+set :application, 'exercism'
+set :repo_url, 'git@github.com:code-R/exercism.io.git'
+
+# Default branch is :master
+set :branch, proc { `git rev-parse --abbrev-ref HEAD`.chomp }.call
+
+# Default deploy_to directory is /var/www/my_app_name
+set :deploy_to, '/home/pramati/projects/exercism/exercism.io'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.